#ifndef ENCLAVE_SAMPLES_PROTOBUF_FUNCTIONS_APP_APP_H_
#define ENCLAVE_SAMPLES_PROTOBUF_FUNCTIONS_APP_APP_H_

#include "./sgx_eid.h"    // sgx_enclave_id_t
#include "./sgx_error.h"  // sgx_status_t

#define ENCLAVE_FILENAME "enclave_protobuf_functions.signed.so"

#if defined(__cplusplus)
extern "C" {
#endif

#if defined(__cplusplus)
}
#endif

#endif  // ENCLAVE_SAMPLES_PROTOBUF_FUNCTIONS_APP_APP_H_
